api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -128,6 +128,9 @@ message PrivateKeyProvider {
   }
 }
 
+// All *DataSource*s in this message support *WatchedFile* when
```

Comment:
Just to double check: it is said all DataSources will be supporting the new WatchedFile but `password` and `ocsp_stable` are not watched & reloaded today? Is the intention to add those as well?

Reply:
This is related to my question above. could we just make everything watched? Or is that too complicated? Otherwise will we fail in all places where it isn't documented and supported?